Update the Note for certificate authentication with a TLS 1.3 connection in windows_winrm.rst

mahoutukaisali opened the pull request:
Since certificate authentication with a TLS 1.3 connection works using Ansible Automation Platform 2.4, updating the Note.

mahoutukaisali committed:
Update windows_winrm.rst 

Since certificate authentication with a TLS 1.3 connection works using Ansible Automation Platform 2.4, updating the Note.

jborean93 reviewed (commented):
Thanks for updating this. I would recommend removing the requests version as it’s more urllib3 where the fix was included

mahoutukaisali committed:
Delete requests python package from Note for certificate authentication with a TLS 1.3 connection in windows_winrm.rst

Delete the requests python package from the mandatory packages since the fix is only included in urllib3 package.

mahoutukaisali commented:
@jborean93 Thanks for the comment. I've deleted `requests` python package from the mandatory.
